app/assets/javascripts/govuk_publishing_components/accessibility-test.js

```diff
@@ -62,6 +95,29 @@
     )
   }
 
+  var _renderIncompleteWarnings = function (incompleteWarnings) {
+    var warningsWrapper = '<div class="component-guide-preview axe-incomplete" data-content="Accessibility Issues: Need Manual Check"></div>'
+
+    incompleteWarnings.forEach(function (warning) {
+      warning.selectors.forEach(function (selector) {
+        var applicableFixtureSelector = selector[0].split('.component-guide-preview', 1) + '.component-guide-preview'
```

Comment:
If we can avoid referencing the guides' presentation classes this would make this module useful elsewhere.